core/templates/services/interaction-specs.service.ts

```diff
@@ -20,10 +20,11 @@ import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
 import { InteractionSpecsConstants } from 'pages/interaction-specs.constants';
+import { InteractionId } from 'interactions/interaction-defs';
 
 @Injectable({providedIn: 'root'})
 export class InteractionSpecsService {
-  isInteractionTrainable(interactionId: string): boolean {
+  isInteractionTrainable(interactionId: InteractionId): boolean {
     const interactionSpecs =
       InteractionSpecsConstants.INTERACTION_SPECS[interactionId];
     if (!interactionSpecs) {
```

Comment:
Do we still need this check?

Reply:
I don't think so, because we have now defined the type for interactionId. As I am a new contributor here, I was not sure to remove this. I'm making a commit to remove the check.